tasks/st1105_primary_server_port_check.rb

```diff
@@ -32,8 +32,12 @@ def port_test(dest, port)
   true
 end
 
+deprecation_msg = "This task is deprecated and will be removed in a future release. Please see this module's README for more information."
 results_json = []
+deprecation = deprecation_msg
```

Comment:
im not sure why you are creating two variables for this, i would just put a json formatted hash into "deprecation" directly